### PR TITLE
asciidoctor: 2.0.10 -> 2.0.12

### PR DESCRIPTION
**Pull request by dcbaker**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/typesetting/asciidoctor/gemset.nix [hunk 1]
```diff
@@ -421,3 +450,4 @@
     version = "1.5.1";
   };
 }
+
```
Comment (SuperSandro2000) [inline pkgs/tools/typesetting/asciidoctor/gemset.nix:453]: Please remove this extra final new line.

Comment (dcbaker) [reply pkgs/tools/typesetting/asciidoctor/gemset.nix]: done.

**Comment by SuperSandro2000** [comment]
```
 results/asciidoctor/bin/asciidoctor
Traceback (most recent call last):
        9: from results/asciidoctor/bin/asciidoctor:18:in `<main>'
        8: from /nix/store/0si882h9k1wv53k8xs95b7zlg8alj57i-bundler-2.1.4/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler.rb:149:in `setup'
        7: from /nix/store/0si882h9k1wv53k8xs95b7zlg8alj57i-bundler-2.1.4/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:20:in `setup'
        6: from /nix/store/0si882h9k1wv53k8xs95b7zlg8alj57i-bundler-2.1.4/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:101:in `block in definition_method'
        5: from /nix/store/0si882h9k1wv53k8xs95b7zlg8alj57i-bundler-2.1.4/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/definition.rb:226:in `requested_specs'
        4: from /nix/store/0si882h9k1wv53k8xs95b7zlg8alj57i-bundler-2.1.4/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/definition.rb:237:in `specs_for'
        3: from /nix/store/0si882h9k1wv53k8xs95b7zlg8alj57i-bundler-2.1.4/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/definition.rb:170:in `specs'
        2: from /nix/store/0si882h9k1wv53k8xs95b7zlg8alj57i-bundler-2.1.4/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/spec_set.rb:80:in `materialize'
        1: from /nix/store/0si882h9k1wv53k8xs95b7zlg8alj57i-bundler-2.1.4/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/spec_set.rb:80:in `map!'
/nix/store/0si882h9k1wv53k8xs95b7zlg8alj57i-bundler-2.1.4/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/spec_set.rb:86:in `block in materialize': Could not find racc-1.5.2 in any of the sources (Bundler::GemNotFound)
```

**Comment by dcbaker** [comment]
I can't reproduce, i've built it locally and used it to build documentation (that doesn't work with the version currently in nix),.

**Comment by SuperSandro2000** [comment]
This is not happening on master. I am not that familiar with ruby in nixpkgs.

**Pull request merged**